frontend/src/components/nodes/NodesMap.tsx

```diff
@@ -36,83 +33,38 @@ interface State {
 class NodesMap extends React.Component<State> {
   state: State = {
     nodesData: [],
-    nodesType: "validators",
+    nodesType: "",
```

Comment:
1. Why?
2. What is the semantics of "" node type? It looks cryptic to me

Reply:
because the validators or onlinenodes data will fetching for a while and it will be weird  if nothing happen when the validators are selected. But then you click each of them instead, it will be correct to display.

Reply:
sorry to say this, but I tried to add componentDidUpdate and shoudComponentUpdate method here, but it still load too much data and can not be control. context component gives data first time is empty and componentdidmount is over. After only click button will trigger setstate method to new data. @frol

Reply:
Well, I don't have time to dive into it this week, so I suggest we merge it as is now and address it in the next iteration.